sites/all/themes/orwell/sass/component/_linkboxes.scss

```diff
@@ -44,16 +44,15 @@ $linkbox__height--medium: 250px;
         max-height: $linkbox__height--medium / 3;
       }
     }
-
   }
 
```

Comment:
Har gjort fonten lidt større, for det virker som om farven er bestemt af noget med specifikt kode, som får mig til at tænke at det ikke bare skal ændres. Hvis fonten er større er kontrasten egentlig fint, men det er ikke en responsiv løsning - altså på en mindre skærm bliver teksten jo mindre med `rem`.

<img width="1106" alt="Screenshot 2020-05-07 at 08 20 47" src="https://user-images.githubusercontent.com/15377965/81268234-fd00fe80-9047-11ea-8d38-09720cee045e.png">